internal/provider/generate.go

```diff
@@ -494,7 +496,8 @@ provider %[1]q {
 		return nil, err
 	}
 
-	tfBin, err := tfinstall.Find(ctx, tfinstall.ExactVersion("0.13.2", tmpDir))
+	g.infof("getting Terraform binary")
+	tfBin, err := tfinstall.Find(ctx, tfinstall.ExactVersion("1.0.1", tmpDir))
```

Comment:
Not sure that it matters here, but protocol 6 is only compatible with TF 1.0.3, and I think there were some bugfixes in 1.0.5 so it might be worth bumping to the latest version 1.0.5